### ISSUE TITLE
How to use it?README is not clear

### ISSUE THREAD

**Issue by Pahkoo** [margelo/react-native-graph#109]
1.usePriceHistory() Where does it come from？
2.LineGraph component ’point‘ property supports incoming data types
3.What specific features are available for the LineGraph component in total？

**Comment by naftalimurgor** [comment]
I'm stuck at the same, is `usePriceHistory(args)` provided by the library? someone please clarify. If not how's this hook structured to return valid `points` prop?

**Comment by camchis** [comment]
`usePriceHistory()` is a custom hook just used as an example, which would return `GraphPoint[]`

**Comment by seelentov** [comment]
`    points: GraphPoint[];`
```
export interface GraphPoint {
    value: number;
    date: Date;
}
```

example:
`points = [{"date": 2024-08-28T19:00:00.000Z, "value": 897.2629999999999}, {"date": 2024-08-29T19:00:00.000Z, "value": 896.959}, {"date": 2024-08-30T19:00:00.000Z, "value": 891.172}, {"date": 2024-09-02T19:00:00.000Z, "value": 880.2819999999999}, {"date": 2024-09-03T19:00:00.000Z, "value": 863.4789999999999}, {"date": 2024-09-04T19:00:00.000Z, "value": 862.935}, {"date": 2024-09-05T19:00:00.000Z, "value": 868.665}, {"date": 2024-09-06T19:00:00.000Z, "value": 874.389}]`

**Comment by naftalimurgor** [comment]
Fair point, How about using custom labels like days of the week instead of Date?

**Comment by seelentov** [comment]
> Fair point, How about using custom labels like days of the week instead of Date?

I think its get more functionality to module. Nothing prevents you from making the dates flat and configuring their output in the component configuration

**Comment by OisinSlevin** [comment]
its still not remotely clear how to use this, the read me page basically gives you linegraph points={data}
can how do I use strings vs values for example or xy plots ?

**Comment by huytdps13400** [comment]
Implemented in #136.

The Usage section no longer references the undefined `usePriceHistory()` hook. It now contains a complete typed `GraphPoint[]` example and explicitly explains that the library does not fetch data, that callers provide points from their own API/state, and that each point uses a `Date` x-value plus a numeric y-value in chronological order.

Prettier, Jest, TypeScript, ESLint, the package build, and the Expo web export all pass.
